jenkins/common.groovy

```diff
@@ -8,6 +8,7 @@ DRUPAL_ADDRESSES = [
   'vagovdev'    : 'http://internal-dsva-vagov-dev-cms-812329399.us-gov-west-1.elb.amazonaws.com',
   'vagovstaging': 'http://internal-dsva-vagov-staging-cms-1188006.us-gov-west-1.elb.amazonaws.com',
   'vagovprod'   : 'http://internal-dsva-vagov-prod-cms-2000800896.us-gov-west-1.elb.amazonaws.com',
+  'sandbox'     : 'https://cms-vets-website-branch-builds-lo9uhqj18nwixunsjadvjsynuni7kk1u.ci.cms.va.gov',
```

Comment:
```suggestion
  // This is a permanent environment maintained by the CMS team specifically for branch builds. It is refreshed to match PROD every 24 hours at 3am ET. Ask in #cms-support if there are issues with this environment. NOTE: Requests to this environment are significantly faster than PROD due to a local database instead of remote (RDS) like PROD CMS has. 
  'sandbox'     : 'https://cms-vets-website-branch-builds-lo9uhqj18nwixunsjadvjsynuni7kk1u.ci.cms.va.gov',
```